apps/site/.eslintrc.cjs

```diff
@@ -30,69 +30,39 @@ module.exports = {
             message:
               "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
           },
-          {
-            name: "@mui/material/Link",
-            message:
-              "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
-          },
           {
             name: "next",
             importNames: ["Link"],
             message:
               "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
           },
           {
-            name: "next/link",
+            name: "next/link.js",
             message:
               "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
           },
           {
             name: "@mui/material",
-            importNames: ["Button", "TextField", "Alert"],
+            importNames: ["Alert", "Button", "Popover", "TextField"],
             message:
               "Please use the custom wrapper component in src/component instead.",
           },
-          {
-            name: "@mui/material/Button",
```

Comment:
Can we keep these specific ones still, banning `@mui/material/Button*` etc? 

If someone imports `@mui/material/Button.js` and doesn't notice the linting error in their editor (or the editor isn't configured properly), they are going to get a linting error in CI which tells them to use `@mui/material`, and once they fix that get _another_ one that tells them to use the custom component, which feels like it could waste time.

With the current change we are hiding an error behind another error, when we have the ability to surface the correct fix immediately.